src/api/__specs__/campaigns-spec.js

```diff
@@ -5,12 +5,21 @@ import * as getJSONP from '../../lib/getJSONP';
 describe('campaigns', () => {
   const callback = sinon.spy();
 
-  beforeEach(() => {
+  before(() => {
     sinon.stub(getJSONP, 'default');
   });
 
-  afterEach(() => {
+  after(() => {
     getJSONP.default.restore();
+  })
+
+  afterEach(() => {
+    try{
+      getJSONP.default.reset();
+    }catch(e){
+      console.error("Failed to reset jsonp, likely timeout issues", e.message);
```

Comment:
I thought it might be the lodash _.defer method but that is stubbed. When I dig down the setTimeouts appear to be what causes the flickers so I'm not sure how to improve on those. Jest's timeFaker does shout out at me.
From what I can see there is a race condition where the timeout will eventually return and try to restore default but it has already been restored. Not catching the error fails the whole test suite so this is a cludge until a better solution is found